census_rm_ops_ui/views/postcode_search.py

```diff
@@ -0,0 +1,33 @@
+from flask import Blueprint, current_app, render_template, request
+
+from census_rm_ops_ui.controllers import case_controller
+
+ADDRESS_SUMMARY_FIELDS = ('organisationName',
```

Comment:
I don't think this is meeting the acceptance criteria, although there are far too many columns IMHO

Reply:
Should be matching the AC now